glsl/color.frag

```diff
@@ -10,8 +22,11 @@ void main()
 
 	if (EnableDepthPreview)
 	{
-		// Front of the depth buffer is at 0, but we want to render it as bright
-		gl_FragColor = vec4(vec3(1.0 - gl_FragDepth), 1.0);
+		float x = 1.0 - gl_FragDepth;
+		float r = clamp(jet_r(x), 0.0, 1.0);
+		float g = clamp(jet_g(x), 0.0, 1.0);
+		float b = clamp(jet_b(x), 0.0, 1.0);
```

Comment:
Pretty sure all the numbers are right.  I convert from NDC to window coords in the line above where the diff starts reporting.